site/posts/python-tutorial-faq.ipynb

```diff
@@ -0,0 +1,249 @@
+    "\n",
+    "------"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Q.** Are there any good guidelines on when to make something a function vs an object method?\n",
+    "\n",
+    "**A.** If you have a collection of functions and those functions all take the mostly same arguments, it is good practice to create an object that holds those shared arguments. We will cover more of this moving forward.\n",
+    "\n",
+    "------"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Q.** Can you talk more about how python manages memory when we read data files or create variables within a function?\n",
+    "\n",
+    "**A.** Memory management is done through a “garbage collection system” and “instance counting” in Python. The first time you create something in Python, memory is allocated for that thing. Every time you access that variable, it counts how many times there are pointers to that object. Once something is no longer pointed to, Python deletes it from memory.\n",
+    "\n",
+    "------"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Q.** In zsh, after running `conda init`, I get an error saying conda is not found...even if I start new zsh shell.  What should I do?\n",
+    "\n",
+    "**A.** You should be able to get conda to work in zsh directly, if you are comfortable with zsh.  It turns out that to do this all you need to do is look inside your \"~/.bash_profile\" file for a section of code that starts with the line:\n",
+    "`# >>> conda initialize >>>`\n",
+    " \n",
+    "and ends with the line:\n",
+    "`# <<< conda initialize <<<`\n",
+    " \n",
+    "If you copy these lines and paste them into a file called \"~/.zshrc\", then conda will work with zsh. \n",
+    "\n",
+    "------"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Some other tips for success:**\n",
+    "- For non mac users, git and conda have 2 different shell environment\n",
+    "- You need to use a text editor - for example nano, vim, or nedit. If you are unfamiliar with text editors pick one \n",
+    "- If the commands at the bottom of the screen are hidden by your Zoom toolbar or meeting controls, you need to change the Zoom settings on your display.\n",
+    "- Here is a video about installing miniconda on windows 10: Install Miniconda and Python 3.7 on Windows 10\n",
+    "- Here is a video about installing anaconda on windows 10: Install Anaconda Python, Jupyter Notebook And Spyder on Windows 10\n",
+    "- Here is a video about installing git and configure git and github on windows: How to Install and Configure Git and GitHub on Windows "
```

Comment:
@jukent, you mention a couple of videos in this section, but the links are currently missing... Do you plan to add them later?

Reply:
Good catch! Added